spec/functions/try_get_value_spec.rb

```diff
@@ -27,22 +27,6 @@
     it 'should exist' do
       is_expected.not_to eq(nil)
     end
-
-    it 'should be able to return a single value' do
-      is_expected.to run.with_params('test').and_return('test')
-    end
```

Comment:
the purpose of this function is to do a lookup on nested data structure. passing string where nested data structure is expected is just wrong and dig() will fail explicitly in this case.